### PR TITLE
Add Amazon Dynamodb connector configuration

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+[[custom_message_formatters]]
+class="org.apache.synapse.commons.json.JsonStreamFormatter"
+content_type = "application/x-amz-json-1.0"
+
+[[custom_message_builders]]
+class="org.apache.synapse.commons.json.JsonStreamBuilder"
+content_type = "application/x-amz-json-1.0"
+```
+
+If you are using **EI 6**, you can enable this property by following Axis2 configurations are added and enabled in the **<PRODUCT_HOME>\repository\conf\axis2\axis2.xml** file.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:25]:
```suggestion
If you are using **EI 6**, you can enable this property by doing the following Axis2 configurations in the **<PRODUCT_HOME>\repository\conf\axis2\axis2.xml** file.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Amazon DynamoDB Connector allows you to access the [Amazon DynamoDB REST API](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.API.html) through WSO2 EI. 
+
+Amazon DynamoDB makes it simple and cost-effective to store and retrieve any amount of data, as well as serve any level of request traffic.It uses a NoSQL database model, which is nonrelational, allowing documents, graphs and columnar among its data models.
+
+## Configuring message builders/formatters**
+
+Before you start configuring the  Amazon DynamoDB connector, you also need WSO2 MI, and we refer to that location as `<PRODUCT_HOME>`.
+
+In this example we will be using specific message builders/formatters configuration needs to be enabled in the product as shown below before starting the WSO2 EI service.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:11]:
```suggestion
Specific message builders/formatters configuration needs to be enabled in the product as shown below before starting the WSO2 EI service.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Setting up the Amazon DynamoDB Connector 
+
+Amazon DynamoDB Connector allows you to access the [Amazon DynamoDB REST API](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.API.html) through WSO2 EI. 
+
+Amazon DynamoDB makes it simple and cost-effective to store and retrieve any amount of data, as well as serve any level of request traffic.It uses a NoSQL database model, which is nonrelational, allowing documents, graphs and columnar among its data models.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:5]:
```suggestion
Amazon DynamoDB makes it simple and cost-effective to store and retrieve any amount of data, as well as serve any level of request traffic. It uses a NoSQL database model, which is non-relational, allowing documents, graphs, and columnar among its data models.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+```
+<messageBuilder contentType="application/x-amz-json-1.0"
+class="org.apache.synapse.commons.json.JsonStreamBuilder"/>
+```
+
+> **Note**: If you want to perform blocking invocations, ensure that the above builder and formatter are added and enabled in the **<PRODUCT_HOME>\repository\conf\axis2\axis2_blocking_client.xml** file.
+
+## Setting up the AWS Account and DynamoDB Environment
+
+Please follow the steps mentioned in the [Setting up the Amazon Lambda Environment](../amazonlambda-connector/setting-up-amazonlambda.md) document in order to create a Amazon account and obtain access key id and secret access key.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:43]:
```suggestion
Please follow the steps mentioned in the [Setting up the Amazon Lambda Environment](../amazonlambda-connector/setting-up-amazonlambda.md) document in order to create an Amazon account and obtain the access key id and secret access key.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Please find the following steps to navigate in to the Amazon DynamoDB using the AWS account.
+
+1. Sign in to the AWS Management Console and search **Database** section under **Services**.
+
+<img src="../../../../assets/img/connectors/dynamodb-aws-console.png" title="Amazon Dynamodb aws console" width="600" alt="Amazon Dynamodb aws console"/> 
+
+2. You can see the following operations and sub operations, outputs in the AWS DynamoDB console as shown bellow.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:51]:
```suggestion
2. You can see the following operations and sub operations. The output in the AWS DynamoDB console are shown below.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:25]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:11]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:5]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:43]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md (hunk 5 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-configuration.md:51]: Updated.

**Pull request merged**